inject-java/src/test/java/io/micronaut/docs/replaces/MockBookService.java

```diff
@@ -31,7 +31,6 @@
  */
 // tag::class[]
 @Replaces(JdbcBookService.class) // <1>
-@Requires(env = Environment.TEST) // <2>
```

Comment:
Should this have been deleted?

Reply:
Yeah - the class in the docs is defined in `src/test`. Since the test environment is enabled by default there, I thought it would cause users to believe that the requires is necessary